src/api/spec/routing/api_matcher_spec.rb

```diff
@@ -3,6 +3,15 @@
 RSpec.describe 'APIMatcher' do
   it { expect(get('/distributions?format=xml')).to route_to(controller: 'distributions', action: 'index', format: 'xml') }
 
+  context 'old person token routes get properly directed' do
```

Comment:
Hm, why do you test this?

Reply:
To be sure that I don't break this when changing the routes.

Reply:
Initially I wanted to use ` resources ` in the routes, but I dropped that. By now the new route is tested. So I dropped the test now.